CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 - Fixed napoleon handling of numpy docstrings with no specified return type.
 
+- Fixed crash when documenting ParamSpecArgs
```

Comment:
This is not the correct place for the changelog entry: 1.21.7 and 1.21.8 were already released. Though the changelog on master looks a bit weird -- there is no entry for 1.21.8 and there is an entry for 1.22.0... Maybe we can just leave it to @gaborbernat to fix it.

Reply:
Sorry - I thought the topmost entry was the upcoming release. I didn't check the version numers 😕

Reply:
Well you don't seem to be the only one who got confused.

Reply:
Maybe move this into a new entry for 1.21.9 and then I'll merge it.